src/test/run-pass/signal-exit-status.rs

```diff
@@ -18,7 +18,7 @@ pub fn main() {
     let args: Vec<String> = env::args().collect();
     if args.len() >= 2 && args[1] == "signal" {
         // Raise a segfault.
-        unsafe { *(0 as *mut isize) = 0; }
+        unsafe { *(1 as *mut isize) = 0; }
```

Comment:
Note to anyone who might be looking at this: #36129 has this change, but it's not merged yet.